### PR TITLE
Fix/circ joint goal feature

### PR DESCRIPTION
**Pull request by martiniil**
Circ goal can as well be given as joint state.

### CODE REVIEW

File: pilz_robot_programming/test/integrationtests/tst_api_cmd_conversion.py [hunk 1]
```diff
@@ -551,6 +546,29 @@ def test_circ_cmd_convert_interim(self):
         self._analyze_request_pose(TARGET_LINK_NAME, exp_goal_pose, req)
         self._analyze_request_circ_help_point("interim", exp_help_pose, req)
 
+    def test_circ_cmd_convert_joint(self):
```
Comment (hslusarek) [inline pilz_robot_programming/test/integrationtests/tst_api_cmd_conversion.py:549]: You test the API. But who tests that the newly created request type actually can be executed?

File: pilz_robot_programming/src/pilz_robot_programming/commands.py [hunk 2]
```diff
@@ -458,7 +458,8 @@ class Circ(_BaseCmd):
         The circle can be completely defined by stating a interim `or` an center position.
         However, only one of both should be stated.
 
-    :param goal: The goal of the motion stated in Cartesian space (geometry_msgs/Pose).
+    :param goal: The goal of the motion, which can be given in joint (list of float, in the order of active joints in
```
Comment (hslusarek) [inline pilz_robot_programming/src/pilz_robot_programming/commands.py]: What about the center or interim point? Can they also be stated as joint values?

Comment (hslusarek) [reply pilz_robot_programming/src/pilz_robot_programming/commands.py]: In case it is possible, please add test to check this.

**Review by hslusarek — CHANGES_REQUESTED** [review]

File: pilz_robot_programming/src/pilz_robot_programming/commands.py (hunk 2 above)
Comment (martiniil) [reply pilz_robot_programming/src/pilz_robot_programming/commands.py]: I agree that it is inconsistent to introduce the joint state option only for the goal.
Reverting this change ...

File: pilz_robot_programming/test/integrationtests/tst_api_cmd_conversion.py (hunk 1 above)
Comment (martiniil) [reply pilz_robot_programming/test/integrationtests/tst_api_cmd_conversion.py:549]: Not relevant any more, since we do not officially include the feature.

**Pull request merged**